src/azure-cli/HISTORY.rst

```diff
@@ -24,6 +24,10 @@ Release History
 
 * Fix #6371: Support filename and environment variable completion in Bash
 
+**ACR**
```

Comment:
Can you add ACR entry to the history log in alphabetic order?

Reply:
Sure, done